async.js

```diff
@@ -0,0 +1,13 @@
+exports.handleLaunch = function(req, res) {
+  //call to the DB with alexaID
+  User.find({alexaID: req.params.alexaID})
```

Comment:
isnt the relevant field in the db called alexaUserId, not alexaId? given that, this line wouldnt work, right? @JasonArkens17 

on closer look, is this async.js file even being used anywhere? if not, can we remove it?